math_hurdler.py

```diff
@@ -148,7 +149,7 @@ def run(self):
             Color.GREEN,
             (0, 0),
             (ground.get_width(), 0),
-            ground.get_height() / 2
+            int(ground.get_height() / 2)
```

Comment:
It'll also be nice to find out if the division here should be done with //, same as all similar occurrences that follow.

Reply:
Sure. Since I'm converting float values to int, I'm anyway getting rid of vales after decimal. Flooring the values using `//` wouldn't make a difference in the result now. If I floor the values, then I think I don't need to typecast to int. Both ways give the same result, I'm not sure which is the most efficient one.

Reply:
> If I floor the values, then I think I don't need to typecast to int. Both ways give the same result, I'm not sure which is the most efficient one.

Well, not exactly.

```python
>>> 3.0 // 2
1.0
>>> int(3.0 / 2)
1
```

If you need the `type` of the result to be of `int`, then, `int` is the best usage. If you don't care of the type of the output, i.e, can be either `float` or `int`, then, I would suggest using `//` instead.

Reply:
okay, we need `int` type here. In this case, would I still need to floor it?

Reply:
@sourahaa said;
> okay, we need int type here. In this case, would I still need to floor it?

No you don't need to use floor division in that case.